lib/pages/home.dart

```diff
@@ -577,8 +577,8 @@ class HomePageStateData {
     return {
       _fromPlace: fromPlace?.toJson(),
       _toPlace: toPlace?.toJson(),
-      _plan: plan.toJson(),
-      _ad: ad.toJson(),
+      _plan: plan?.toJson(),
+      _ad: ad?.toJson(),
```

Comment:
I forgot during the refactoring to add the null safety back in place.
Original Code in Master
```dart
      _FromPlace: fromPlace != null ? fromPlace.toJson() : null,
      _ToPlace: toPlace != null ? toPlace.toJson() : null,
      _Plan: plan != null ? plan.toJson() : null,
      _Ad: ad != null ? ad.toJson() : null,
```